### DISCUSSION TITLE
Many minor corrections & New Functionality

### DISCUSSION THREAD

**Discussion by pschatzmann** [pschatzmann/arduino-audio-tools#1394] [Announcements]
I noticed that I haven't published any release for a long time. 
This release contains __many error corrections__ and the following new functionality

New Functionality
- Codec L8 and L16
- RIFF file format
- Adpcm codec
- Opus Ogg
- AACDecoderADTS

Pltforms
- Arduino UNO R4
- Nano BLE Sense: I2S Support
- ESP32 Arduino Support for 3.0.0

Integrations
- Faust Support with Examples
- Integration with Pure Data
- New Audio Driver Concept for supporting LyraT, AudioKit and many other codecs

Communication
- Throttle
- VBAN Support
- FTP example

New Classes
- PIDController
- Median Filter
- ChannelSplitOutput

<hr /><em>This discussion was created from the release <a href='https://github.com/pschatzmann/arduino-audio-tools/releases/tag/v0.9.7'>Many minor corrections & New Functionality</a>.</em>

**Comment by KevynWithaY** [comment]
Thank you! You are appreciated and I hope you are well. Looking forward to trying the new additions. Awesome library!
